pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/BaseContext.java

```diff
@@ -217,4 +218,15 @@ default ClientBuilder getPulsarClientBuilder() {
         throw new UnsupportedOperationException("not implemented");
     }
 
+    /**
+     * Get the pre-configured pulsar admin client.
+     *
+     * You can use this admin client to administrate a Pulsar cluster.
+     *
+     * @return The instance of pulsar admin client
+     */
+    default PulsarAdmin getPulsarAdmin() {
+        throw new UnsupportedOperationException("not implemented");
```

Comment:
what about "Not available in this context" ?

Reply:
I wanted to keep it consistent with the messages for e.g. `getPulsarClient` but I'm open to changing too